prow/config/config.yaml

```diff
@@ -7,20 +7,66 @@ prowjob_namespace: ${PROWJOB_NAMESPACE}
 pod_namespace: ${TEST_PODS_NAMESPACE}
 
 # more logs
-log_level: debug
+log_level: info
 
 branch-protection:
   enforce_admins: true # rules apply to admins too!
   orgs:
     tikv:
       repos:
+        client-c:
+          branches:
+            master: &client-c-branch-pt
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "tide"
+                  - "DCO"
+                  - "idc-jenkins-ci/test"
+                strict: false
+              restrictions: *branch-protection_restrictions
+            release-8.5: *client-c-branch-pt
+            release-8.1: *client-c-branch-pt
+            release-7.5: *client-c-branch-pt
+            release-7.1: *client-c-branch-pt
+            release-6.5: *client-c-branch-pt
+            release-6.1: *client-c-branch-pt
```

Comment:
/cc @gengliqi
PTAL this part, the `release-8.5` is prepared for future LTS branch.

Reply:
`strict` is fixed to `true` in the next commit.